package proto_ff;

import "yd_fieldoptions.proto";


message treasureBowltreasureBowl
{
	optional int32 Id = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 level = 2[(yd_fieldoptions.field_cname) = "等级"];
	optional int32 Type = 3[(yd_fieldoptions.field_cname) = "解锁价格类型"];
	optional int32 price = 4[(yd_fieldoptions.field_cname) = "解锁价格"];
	optional int32 returnD = 5[(yd_fieldoptions.field_cname) = "钻石返还比例"];
	optional int32 returnVip = 6[(yd_fieldoptions.field_cname) = "vip经验返还比例"];
	optional int64 returnItemID = 7[(yd_fieldoptions.field_cname) = "特殊物品返还ID"];
	optional int32 returnItem = 8[(yd_fieldoptions.field_cname) = "特殊物品返还比例"];
	optional int64 boxid = 9[(yd_fieldoptions.field_cname) = "解锁奖励"];
	optional int64 skillid = 10[(yd_fieldoptions.field_cname) = "解锁特权技能"];
	optional string skilldes = 11[(yd_fieldoptions.field_cname) = "解锁特权技能", (yd_fieldoptions.field_bufsize) = 60];
}


message Sheet_treasureBowltreasureBowl
{
	repeated treasureBowltreasureBowl treasureBowltreasureBowl_List = 1[(yd_fieldoptions.field_arysize)=20];
}
